src/MessageHandler/LoggedPredisClient.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace M6Web\Component\RedisMessageBroker\MessageHandler;
+
+use Predis\Client as PredisClient;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+
+class LoggedPredisClient
+{
+    use LoggerAwareTrait;
+
+    /** @var PredisClient */
+    private $decorated;
+
+    public function __construct(PredisClient $redisClient, LoggerInterface $logger = null)
```

Comment:
I don't understand why it is allowed here to provide a `null` logger as the only purpose of this class is to log redis calls.

Reply:
good point